src/test/java/com/wikia/webdriver/common/core/helpers/UserWithEmailFactory.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.common.core.helpers;
+
+
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+
+public class UserWithEmailFactory {
+
+  private UserWithEmailFactory() {
+    // no-op
+  }
+
+  private static final Credentials CREDENTIALS = Configuration.getCredentials();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this variable to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1213)